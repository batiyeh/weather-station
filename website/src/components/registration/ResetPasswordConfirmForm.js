import React, { Component } from 'react';
import '../../styles/login.css';

class ResetPasswordConfirmForm extends Component {
    constructor(props){
        super(props);
        this.tokenUrl = '/api/user/reset/' + this.props.token;
    }
    render(){
        return(
            <div className="confirm-container">
                <form className='ResetPasswordForm' action={this.tokenUrl} method='post' >
                    <div className='form-group'>
                        <input id='password' name='password' type='password' class='form-control' placeholder='Password'/>
                    </div>
                    <div className='form-group'>
                        <input id='password2' name='password2' type='password' class='form-control' placeholder='Confirm Password'/>
                    </div>
                    <div className='row'>
                        <div className='col-6'>
                            <button type='submit' className='btn btn-info login-btn'>Reset Password</button>
                        </div>
                    </div>
                </form>
            </div>
        )
    }
}

export default ResetPasswordConfirmForm;