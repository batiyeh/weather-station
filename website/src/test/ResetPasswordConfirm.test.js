import React from 'react'
import ResetPasswordConfirmForm from '../components/registration/ResetPasswordConfirmForm'
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
    .create(<ResetPasswordConfirmForm/>).toJSON();
    expect(tree).toMatchSnapshot();
})