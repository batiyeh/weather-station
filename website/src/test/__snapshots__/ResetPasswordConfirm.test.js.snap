// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="confirm-container"
>
  <h2
    className="login-title"
  >
    Reset Password
  </h2>
  <form
    action="/api/user/reset/undefined"
    className="ResetPasswordForm"
    method="post"
  >
    <div
      className="form-group"
    >
      <input
        class="form-control"
        id="password"
        name="password"
        placeholder="Password"
        type="password"
      />
    </div>
    <div
      className="form-group"
    >
      <input
        class="form-control"
        id="password2"
        name="password2"
        placeholder="Confirm Password"
        type="password"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="col-6 no-padding-left"
      >
        <button
          className="btn btn-default btn-block login-btn"
          type="submit"
        >
          Reset Password
        </button>
      </div>
    </div>
  </form>
</div>
`;
